src/sentry/runner/commands/createuser.py

```diff
@@ -76,24 +77,30 @@ def createuser(email, password, superuser, no_password, no_input):
     if password:
         user.set_password(password)
 
-    user.save()
```

Comment:
How about we just did

```py
user.save(force_update=force_update)
```

With the `--force` addition you've made here (renaming it to `--force-update`)?

Reply:
Done! Good suggestion, thanks! 👍